### ISSUE TITLE
No support for UTF-8 encoding

### ISSUE THREAD

**Issue by prokhor-ozornin** [ajay-kmr/swagger#1]
ApiDocController does not support fetching HTML content from webjars (swagger-ui) with UTF-8, so international characters (like Cyrillic) from swagger annotations descriptions are rendered incorrectly (with question marks - '?')

**Comment by ajay-kmr** [comment]
Hi Prokhor Ozornin,

Please share the complete annotations which you are using at controller's method label.
This is required to replicate this issue.

Thanks

**Comment by prokhor-ozornin** [comment]
The problem in fact is in generated Swagger JSON document - it's not returned in correct encoding.

Here is a fragment of returned JSON:
`{"swagger":"2.0","info":{"description":"API ??? ?????? ? ??????????? ????????","version":"1.0","schemes":["http"],"consumes":["application/json"],"paths":{"/account/{accountUuid}/balance":{"get":{"summary":"????????? ??????????? ??????? ??? ????? ???????????? ? ???????? ???????????????","operationId":"/account/{accountUuid}/balance","produces":["application/json"],"parameters":[{"name":"accountUuid","in":"path","description":"????????????? ????? ????????????","required":true,"type":"string"},{"name":"date","in":"query","description":"???? (???/?????) ???????","required":false,"type":"string"},{"name":"dateFrom","in":"query","description":"????/????? ?????? ?????????? ?????????","required":false,"type":"string"}`
.....

Here is the example of annotation with cyrillic chars:
```
  @ApiOperation(
    nickname = '/account/{accountUuid}/csv',
    httpMethod = 'GET',
    response = Void,
    value = 'Экспорт списка платежей, найденных по заданным критериям, в формат CSV',
    produces = 'application/vnd.ms-excel')
```

Perhaps you should replace
`response.contentType = 'application/json'`
with
`response.contentType = 'application/json; charset=UTF-8'`
in ApiDocController?

**Comment by ajay-kmr** [comment]
Hi Prokhor Ozornin,

Thanks for your feedback.
Necessary changes will be done in upcoming release.

**Comment by ajay-kmr** [comment]
The issue has been resolved.

**Comment by ajay-kmr** [comment]
Issue resolved in version>= 1.0.0
